force-app/main/default/lwc/nksConvNoteViewer/nksConvNoteViewer.css

```diff
@@ -5,3 +5,8 @@
 .conv-note-details {
     min-height: 30em;
 }
+
+.img-reset {
+    --lwc-spacingSmall: 0px;
+    --slds-g-spacing-3: 0.3rem;
+}
```

Comment:
Fungerer dette for slds1? --slds-g-* variablene er jo for global styling som kommer i slds 2. Det er også egentlig ikke noe vi skal skrive over heller 😅

Reply:
oi, nei det har jeg ikke sjekket